### ISSUE TITLE
Creating Personal Time Tracking Pages on Wiki

### ISSUE THREAD

**Issue by CahidArda** [bounswe/bounswe2022group7#41]
It was decided that each team member will have a page for effort tracking seperate from the personal wiki page. A [template](https://github.com/bounswe/bounswe2022group7/wiki/Weekly-Effort-Reports-Template) is prepared ([#31](https://github.com/bounswe/bounswe2022group7/issues/31)). Each team member should do the following:
- Create a page on wiki with the following name format: "Cahid Arda Öz (Effort Tracking)".
- Fill the page according to the [template](https://github.com/bounswe/bounswe2022group7/wiki/Weekly-Effort-Reports-Template).
- Remove the weekly effort tracking section from your personal wiki page.
- Add the hyperlink to your new effort tracking page to the [Weekly Effort Tracking](https://github.com/bounswe/bounswe2022group7/wiki/Weekly-Effort-Tracking) on wiki.

This task should be completed until 14.3.2022. @CahidArda is responsible with tracking.

Checklist:
- [X] @CahidArda 
- [ ] @demet47 
- [ ] @AlicanM 
- [ ] @musasimsekdotdll
- [ ] @erimerkin 
- [ ] @mehmeteminipekdal 
- [ ] @guneyizol 
- [ ] @azizamankenova 
- [ ] @oguzpancuk 
- [ ] @atillaturkmen 
- [ ] @canatakan 
- [ ] @HasanBingolbali

**Comment by guneyizol** [comment]
I created my effort tracking page :+1:

**Comment by canatakan** [comment]
In order not to wait for each member to create their own pages, I have now created the personal time tracking pages for all members and added the links to [the relevant page](https://github.com/bounswe/bounswe2022group7/wiki/Weekly-Effort-Tracking). Now, every member must edit their own weekly time tracking reports as they like. 

After this update, it is each member's own duty to edit their own personal pages and there is no need for checkboxes in this issue as those will just delay the completion of this task.

@CahidArda can now review the created template pages and the links.

**Comment by CahidArda** [comment]
Template and links are added. Closing the issue.
